### PR TITLE
Fix/fmap 43

### PR DESCRIPTION
**Pull request by elkyndiosa**
#### What problem is this solving?

Numbers arrived without decimals, the decimals were added in string format

### CODE REVIEW

File: node/utils/products.ts [hunk 1]
```diff
@@ -51,3 +51,9 @@ export const mapSkus = (skuURL: string) => {
     }, []) ?? []
   )
 }
+const decimalNumber = (num: number): string | number => {
```
Comment (raphaelsaez) [inline node/utils/products.ts]: hey @elkyndiosa why did you changed it for response an string or number?

File: node/utils/products.ts [hunk 2]
```diff
@@ -9,12 +9,12 @@ export const mapProducts = (
 
       return sku !== undefined
     })
-
+    const sellingPrice = decimalNumber(sku?.sellers[0].commertialOffer.Price as number)
```
Comment (raphaelsaez) [inline node/utils/products.ts]: @elkyndiosa please check it, you are calling your method before create it

File: node/utils/products.ts [hunk 3]
```diff
@@ -51,3 +51,9 @@ export const mapSkus = (skuURL: string) => {
     }, []) ?? []
   )
 }
+const decimalNumber = (num: number): string | number => {
+    if(num === undefined) return num
```
Comment (raphaelsaez) [inline node/utils/products.ts]: @elkyndiosa it's a little confuse, if you receive an undefined value, so you gonna return an undefined response, and this doesn't match with your response types. btw: you can easily use `if (!num) return ...`

File: node/utils/products.ts [hunk 4]
```diff
@@ -51,3 +51,9 @@ export const mapSkus = (skuURL: string) => {
...
+    const dec = num.toString().split('.')[1]
```
Comment (raphaelsaez) [inline node/utils/products.ts]: a tip here -> https://williamdurand.fr/2013/06/03/object-calisthenics/#5-one-dot-per-line

File: node/utils/products.ts [hunk 5]
```diff
@@ -51,3 +51,9 @@ export const mapSkus = (skuURL: string) => {
...
+    const len = dec && dec.length > 2 ? dec.length : 2
+    return Number(num).toFixed(len)
```
Comment (raphaelsaez) [inline node/utils/products.ts]: your method is getting a number and returning a string, you could improve it

**Review by raphaelsaez — CHANGES_REQUESTED** [review]
as a general reminder, please check all the warnings and errors from lint please

**Comment by raphaelsaez** [comment]
@elkyndiosa maybe this could help you:
`const currencyLocale = Intl.NumberFormat('en-US');
console.log(currencyLocale.format(number))`

File: node/utils/products.ts (hunk 5 above)
Comment (elkyndiosa) [reply node/utils/products.ts]: Yes, this is the solution. I explain: Javascript by default transform a number with decimals .00 to without decimal, example if one number is 10.00 javascript being a number, shows it without the decimals, by this motive the solution is return string.
IMPORTANT: I tested in the template of the mail and there was not problem at receive this data with a type string. I test with the helpers of handlerbars for check and not harm anything that is implemented.

**Pull request merged**